src/components/common/Buttons/BasicButton.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const BasicButton = props => {
+  const { className, onClick, title } = props;
+  return (
+    <button
+      type="button"
+      className={`btn ${className}`}
+      {...props}
```

Comment:
@abayo-luc, Do we need another button? what do you think using the existing **Confirm** button and customize it according to your needs?